infrastructure/main.tf

```diff
@@ -13,7 +13,7 @@ provider "vault" {
 }
 
 locals {
-  aseName = "${data.terraform_remote_state.core_apps_compute.ase_name[0]}"
+  aseName = "core-compute-${var.env}"
```

Comment:
same as: ccd-user-profile-api-POSTGRES-PASS

Reply:
as a secret? 🤔 not sure...does it matter 😺